packages/grpc-native-core/src/client.js

```diff
@@ -939,9 +939,11 @@ exports.makeClientConstructor = function(methods, serviceName,
       throw new Error('Method names cannot start with $');
     }
     var method_type = common.getMethodType(attrs);
-    var method_func = _.partial(requester_funcs[method_type], attrs.path,
-                                attrs.requestSerialize,
-                                attrs.responseDeserialize);
+    var method_func = function() {
+      return requester_funcs[method_type].apply(this, arguments.concat([
+        attrs.path, attrs.requestSerialize, attrs.responseDeserialize
```

Comment:
I think this is putting those arguments at the end of the final list, instead of at the beginning.